src/main/java/net/sf/jabref/gui/keyboard/KeyBinding.java

```diff
@@ -13,6 +13,7 @@
     BACK("Back", Localization.lang("Back"), "alt LEFT"),
     CLEANUP(
             "Cleanup", Localization.lang("Cleanup entries"), "F8"),
+    CHECK_INTEGRITY("Check integrity", Localization.lang("Check integrity"), "ctrl F8"),
```

Comment:
What was our last consensus about showing menu entries in the hotkey dialog? I think, I opted for using `Localization.lang`, but we might have decided to re-translate these entries. @boceckts Do you remember? I think, we had the discussion. @tobiasdiez Is that documented in our CodeHowTo in the Wiki?